python/dgl/data/graph_serialize.py

```diff
@@ -108,42 +119,55 @@ def save_graphs(filename, g_list, labels=None):
     _CAPI_DGLSaveGraphs(filename, gdata_list, label_dict)
 
 
-def load_graphs(filename, idx_list=None):
+def load_graphs(filename, idx_list=None, ignore_labels=False):
     """
     Load DGLGraphs from file
 
     Parameters
     ----------
     filename: str
-        filename to load DGLGraphs
+        filename to load graphs
     idx_list: list of int
         list of index of graph to be loaded. If not specified, will
         load all graphs from file
+    ignore_labels: bool
+        Whether to ignore the return of labels
 
     Returns
     ----------
-    graph_list: list of immutable DGLGraphs
-    labels: dict of labels stored in file (empty dict returned if no
+
+    graph_list: list of DGLGraphs / DGLHeteroGraph
+    labels(Optional): dict of labels stored in file (empty dict returned if no
     label stored)
 
+    If the file is stored with labels (before DGL 0.4.4) and set ignore_labels=False,
+     it will return graph_list and labels
+    If the file isn't stored with labels, it will only return graph_list

```

Comment:
Why not just checking the version number and returns w/ or w/o labels correspondingly?